python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -155,36 +147,48 @@ async def begin_standby(self, id_data):
 
         self.telemetryLoopTask.cancel()
         self.kiloarcListenerTask.cancel()
-        await self.chillerModel.disconnect()
```

Comment:
If you go into FAULT state before `self.chillerModel` is set then you will never be able to get out of it (the usual transition being FAULT -> STANDBY) because `if if self.chillerModel.alarmPresent` will raise an exception.

Please guard against this by testing if self.chillerModel is None before trying to interact with it. For example:
```
if self.chillerModel is not None and self.chillerModel.alarmPresent:
    alarmlist = ...
```